package.json

```diff
@@ -26,14 +26,16 @@
     "http-errors": "~1.6.3",
     "morgan": "~1.9.1",
     "mysql2": "^2.3.3",
-    "nodemon": "^2.0.15",
     "sequelize": "^6.19.0",
     "sequelize-cli": "6.2.0"
   },
   "devDependencies": {
     "chai": "^4.3.4",
     "jest": "27.0.6",
     "mocha": "^9.1.3",
+    "nodemon": "^2.0.20",
+    "pg": "^8.8.0",
+    "pg-hstore": "^2.3.4",
```

Comment:
this package it's not necessary, npm remove pg-hstore

Reply:
I have problems using mysql at the moment I am using postgres, the packages can be removed at the end when the project is going to be released to production.